main.go

```diff
@@ -75,8 +79,10 @@ func main() {
 	flagSet.StringVar(&leaderElectResourceNamespace, "leader-elect-resource-namespace", "openshift-cluster-machine-approver", "the namespace in which the leader election resource will be created.")
 	flagSet.Parse(os.Args[1:])
 
-	if err := validateAPIGroup(APIGroup); err != nil {
-		klog.Fatalf(err.Error())
+	for _, apiGroup := range APIGroups {
+		if err := validateAPIGroup(apiGroup); err != nil {
```

Comment:
This won't work with group version strings unless the function is updated to support group versions